LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
```

Comment:
@DailyDreaming Could you point us to a UCSC project that have this License that we can take from? If that's the license that you're already using, then @uwcdc could update it to be so. We used the BSD 3-Clause License because that was recommended by our department here at the University of Washington for the software we help develop and I didn't any LICENSE file in braingeneerspy at the time.

Reply:
@DailyDreaming I was looking through the `braingeneers` repos for active projects with your Apache 2.0 license. I am only seeing MIT, BSD 3-clause, and GPL-2 licenses for most of the projects. Could you please provide a copy of the license, so I can resolve this conversation?

Reply:
@lsetiawan @uwcdc We had a discussion about this yesterday and I think [the MIT license](https://opensource.org/license/mit/) is appropriate.  My Apache suggestion is apparently outdated.  >.<

Reply:
Updated the `LICENSE` and `README.md` to MIT